exquery-restxq-specification/restxq-1.0-specification.html

```diff
@@ -713,6 +723,23 @@ <h3>Base URI</h3>
           <!-- TODO if a path annotation is not specified then the remaining URIs apply to the base uri -->
         </p>
       </section>
+      
+      <section id="content-negotiation">
+        <h3>Content Negotiation</h3>
+        <p>Support for <a href="https://httpwg.org/specs/rfc7231.html#content.negotiation">content negotiation</a> ([[RFC7231]]) of formats is indirectly provided by the <a href="#consumes-annotation"><code>%rest:consumes</code></a> and <a href="#produces-annotation"><code>%rest:produces</code></a> annotations.</p>
+        
+        <pre class="example highlight" title="Content Negotiation">
+            (: Combining consumes and produces provides content negotation support :)
+            declare
+              %rest:path("/widgets")
+              %rest:consumes("application/xml", "application/atom+xml")
+              %rest:produces("application/xml", "application/json")
+            function local:widgets() {
+              fn:collection("/db/widgets")/widgets
+            };
+          </pre>
+        <p>The mechanism by which formats are converted are implementation dependent.</p>
```

Comment:
I don't think that is entirely correct, as it is also influenced by the `%output:method` annotation. I would suggest removing this paragraph please.

Reply:
OK; I wanted to clarify that the (automatic) representation of e.g. an XML fragment as a JSON map (such as is supported natively by eXist-db) was outside the scope of the specification.  For now I will remove it as you suggest.